pysat/instruments/methods/testing.py

```diff
@@ -1,4 +1,94 @@
+import os
 import numpy as np
+import pandas as pds
+import pysat
+
+
+def list_files(tag=None, sat_id=None, data_path=None, format_str=None,
+               file_date_range=None, test_dates=None):
+    """Produce a fake list of files spanning three years
+
+    Parameters
+    ----------
+    tag : (str)
+        pysat instrument tag (default=None)
+    sat_id : (str)
+        pysat satellite ID tag (default=None)
+    data_path : (str)
+        pysat data path (default=None)
+    format_str : (str)
+        file format string (default=None)
+    file_date_range : (pds.date_range)
+        File date range. The default mode generates a list of 3 years of daily
+        files (1 year back, 2 years forward) based on the test_dates passed
+        through below.  Otherwise, accepts a range of files specified by the
+        user.
+        (default=None)
+    test_dates : (dt.datetime)
```

Comment:
How about we leave the code as is and just update the docstring to reflect the input is a dictionary of _test_dates, required by every pysat.Instrument?

Reply:
updated the docstring.  ready for re-review